ext/zend_test/tests/gen_stub_test_02.phpt

```diff
@@ -0,0 +1,18 @@
+--TEST--
+gen_stub.php: Test that attributes are applied to methods
+--EXTENSIONS--
+zend_test
+--FILE--
+<?php
+
+class Foo {
+    use _ZendTestTrait;
+}
+
+$reflection = new ReflectionMethod(Foo::class, "testMethod");
+
+var_dump($reflection->getAttributes()[0]->getName());
+
+?>
+--EXPECTF--
+string(%d) "ZendTestAttribute"
```

Comment:
```suggestion
var_dump($reflection->getAttributes()[0]->newInstance());

?>
--EXPECTF--
object(ZendTestAttribute)#%d (0) {
}
```

To verify that the actual attribute class is applied (cf. `ext/zend_test/tests/attribute_arguments.phpt`).

Reply:
Fixed!